packages/help-center/src/components/help-center-more-resources.tsx

```diff
@@ -117,31 +71,19 @@ export const HelpCenterMoreResources = () => {
 						</a>
 					</div>
 				</li>
-				{ showWhatsNewItem && (
-					<li className="help-center-more-resources__resource-item help-center-link__item">
-						<div className="help-center-more-resources__resource-cell help-center-link__cell">
-							<Button
-								variant="link"
-								onClick={ handleWhatsNewClick }
-								className="help-center-more-resources__new-releases"
-							>
-								<Icon icon={ <NewReleases /> } size={ 24 } />
-								<span>{ __( "What's New", __i18n_text_domain__ ) }</span>
-								{ showWhatsNewDot && (
```

Comment:
This is a pretty neat feature, its a shame to lose it!

Reply:
Is that a notification for when we make updates?

Reply:
yeah